example-project/build.gradle

```diff
@@ -0,0 +1,113 @@
+// but usually most people write their Hadoop DSL in a separate file (or multiple files).
+apply from: 'src/main/gradle/workflows.gradle'
+
+dependencies {
+  // Dependencies for Hadoop Java jobs
+  compile 'org.apache.hadoop:hadoop-common:2.6.1'
+  compile 'org.apache.hadoop:hadoop-mapreduce-client-core:2.6.1'
+
+  // Dependencies to use TestNG for unit tests
+  testCompile 'org.testng:testng:6.8.8'
+
+  // Dependencies for Hadoop mini-cluster based unit tests
+  testCompile 'org.apache.hadoop:hadoop-common:2.6.1:tests'
+  testCompile 'org.apache.hadoop:hadoop-hdfs:2.6.1:tests'
+  testCompile 'org.apache.hadoop:hadoop-mapreduce-client-common:2.6.1'
+  testCompile 'org.apache.hadoop:hadoop-mapreduce-client-hs:2.6.1'
+  testCompile 'org.apache.hadoop:hadoop-mapreduce-client-jobclient:2.6.1'
+  testCompile 'org.apache.hadoop:hadoop-mapreduce-client-jobclient:2.6.1:tests'
+  testCompile 'org.apache.hadoop:hadoop-yarn-server-tests:2.6.1:tests'
+}
+
+// Use TestNG for unit tests
+test {
+  useTestNG()
+}
+
+// Declare that build depends on the Hadoop zips, which depends on compiling the Hadoop DSL. Adding
+// this is what triggers the compilation of the Hadoop DSL as a part of your build.
+startHadoopZips.dependsOn buildAzkabanFlows
+build.dependsOn buildHadoopZips
+
+// Now declare the hadoopZip block to easily to build zip artifacts that can be uploaded to Azkaban.
+hadoopZip {
+  libPath = "lib"  // Causes the dependencies to be placed in the "lib" folder inside each zip
+
+  // Use `base` to declare files common to all the zips you want to declare (as you might have
+  // different zips for different Hadoop clusters or Azkaban instances, for whatever reason). If
+  // you just need a single zip, there's no point in using `base` and you can skip it.
+  base {
+    from("src/main/bash") {
+      into "bash"   // Bash scripts will be in the "bash" subdirectory in the zip
+    }
+    from("src/main/hive") {
+      into "hive"  // Hive scripts will be in the "hive" subdirectory in the zip
+    }
+    from("src/main/pig") {
+      into "pig"   // Pig scripts will be in the "pig" subdirectory in the zip
+    }
+    // Your workflows.gradle file specifies that the compiled Hadoop DSL files will be placed in
+    // the "azkaban" folder. Add them to your zips using closure syntax, since the compiled files
+    // haven't been generated yet.
+    from { fileTree("azkaban").files }
+   }
+
+  // Note that dependencies the hadoopRuntime dependency configuration (which inherits from the
+  // runtime configuration) are automatically included in all zips (including the project jar).
+
+  // Now declare each of your zips, giving them unique names. In this (completely contrived) case
+  // we will run the word count jobs on different text files for our dev and prod clusters.
+  zip("azkabanDev") {
+    from("resources-dev") {
+      into "text"
+    }
+  }
+
+  zip("azkabanProd") {
+    from("resources-prod") {
+      into "text"
+    }
+  }
+}
```

Comment:
Lets have a dir structure for resources by having resources/dev, resources/prod etc